### ISSUE TITLE
Unable to update expired key

### ISSUE THREAD

**Issue by kian** [keybase/keybase-issues#1707]
I set up Keybase a while back, and I recently learned my key had expired.  I then updated the expiry and pushed the key to pgp.mit.edu  (see [link](https://pgp.mit.edu/pks/lookup?op=vindex&search=0x68DD9D247C8EA90C)).

```
$ gpg2 --list-keys
/Users/kian/.gnupg/pubring.kbx
------------------------------
pub   dsa1024/7C8EA90C 2009-12-22 [expires: 2016-11-24]
uid       [ unknown] Kian Mohageri <kian.mohageri@paperg.com>
sub   elg2048/50434F73 2009-12-22 [expires: 2016-11-24]
```
I then tried to push my updated key via the CLI and received an error:

```
$ keybase push
error: Subkey c2c95ca47a06d3e5305e939168dd9d247c8ea90c expired at 1419145435 but we checked for time 1437762624
```

Other operations seem to fail the same way.
```
$ keybase pull
info: Updated file: /Users/kian/.cache/keybase/session.json
error: Subkey c2c95ca47a06d3e5305e939168dd9d247c8ea90c expired at 1419145435 but we checked for time 1437762624

$ keybase list-tracking
error: Subkey c2c95ca47a06d3e5305e939168dd9d247c8ea90c expired at 1419145435 but we checked for time 1437762624

$ keybase status
error: Subkey c2c95ca47a06d3e5305e939168dd9d247c8ea90c expired at 1419145435 but we checked for time 1437762624
```

Notes:

 - Fresh installation, so this shouldn't be caused by stale local state (as far as I can tell)
 - Running 0.8.12 on OSX

What is the correct way to update keys on Keybase, once they have expired?

**Comment by maxtaco** [comment]
Fix should be live in v0.8.14.  LMK if you still have an issue. `keybase push --update` should now work.

**Comment by kian** [comment]
Verified this is now fixed.  Thanks @maxtaco for the quick turnaround!
